docs/tasks/federation/Policy.rego

```diff
@@ -0,0 +1,32 @@
+package kubernetes.placement
```

Comment:
Currently, Rego is the only language supported by OPA. I will add a comment about this.

I'll update the policy to explain how it works and provide a link to more documentation.